spec/classes/foreman_proxy__plugin__ansible_spec.rb

```diff
@@ -82,6 +110,14 @@
           ])
         end
       end
+
+      describe 'with disabled ansible-runner install' do
+        let :params do
+          { install_runner: false }
+        end
+
+        it { should_not contain_package('ansible-runner') }
```

Comment:
Could you add the repo checks here as well (that they shouldn't be present)? You can also add `it { should_not contain_class('foreman_proxy::plugin::ansible::runner') }`

Reply:
I did both :+1: